meetings/2022-03-01.md

```diff
@@ -0,0 +1,134 @@
+* ePa: I see it as good improvement. We may need to do one more, hopefully final, iteration when we have some examples going beyond Data Registrations. So Generic authorizations for data browser like apps and admin level / trusted authorizations.
+* JB: Section 7 of the spec is called Data Authorization. It broadly describes authorization flows. Does anyone have any suggestions for new good name.
+* LD: Why not *Authorization Flows* as you just referred to it.
+* JB: I would also consider Authorization Patterns and *Authorization Scenarios*
+
+ACTION: JB: to make PR with renaming and change section 7 title.
+
+### Source of truth for access control
+
+* WT: Cf. discussion LDB & WT: Are these the consents/grants in the registries, or the decisions of the AS? In particular: can a (UMA) AS overrule AA consents/grants, either temporary or by changing the ACPs/ACRs?
+* WT: Laurens and I discussed AA AS and their interaction. What is the source of truth for the access control. If Access Grants set by AA it would be weird if AS couldn't overwrite it.
+* EP: I see AS associated with RS, on the other hand AA is associated with User (RO sometimes also End-user). We would like to support applying authorizations across multiple RSs each possibly having different AS.
+* JB: I agree. Access Authorizations and Data Authorizations in some cases will still apply to new Resource Servers.
+* LD: Do you need UMA style AS at this point or AA which can configure UMA style ASs. I think you could make UMA AS understanding interop. It would also need to have access to Access Grants and registrations. Do we imagine ASs aware of interop spec?
+* WT: I lost Laurens when he spoke about UMA. I don't know why it would be neccessary for AS to know grants and consents.
+* elf Pavlik: Great topic for wednesday authorization panel. the way i see it is that authz agent needs to be able to do access authorizations and access grants. authz server needs to be able to access the access grants applicable to the RS it is associated with. then based on those generate ACRs / ACLs. keeps resource server from having to know anything about data grants. feels like an attractive separation of concerns. i think the authz server that implements UMA would need to be aware of access / data grants - then whatever the resource server is using (ACP/WAC/etc).
+* WT: why would AS need to handle the dual roles - exchange of tokens / grant awareness
+* elf Pavlik: it's a matter of what else we want to put on the authorization server in addition to exchange of tokens. ... seems to me nice to not put more responsibilities on the authorization agent. since the resource server and authorization server have a pre-established relationship it seems straightforward that the RS allows the AS to write all of the ACRs / ACLs. In the approach i see only the AS would need to write the ACRs / ACLs (i.e. control mode).
```

Comment:
```suggestion
* elf Pavlik: it's a matter of what else we want to put on the authorization server in addition to exchange of tokens. ... seems to me nice to not put more responsibilities on the authorization agent. since the resource server and authorization server have a pre-established relationship it seems straightforward that the RS allows the AS to write all of the ACRs / ACLs. In the approach i see, only the AS would need to write the ACRs / ACLs (i.e., control mode).
```